pylint/checkers/typecheck.py

```diff
@@ -1055,15 +1055,7 @@ def visit_attribute(self, node: nodes.Attribute) -> None:
                 return
 
             try:
-                if not [
-                    n
-                    for n in owner.getattr(node.attrname)
-                    if not isinstance(n.statement(future=True), nodes.AugAssign)
-                ]:
-                    missingattr.add((owner, name))
-                    continue
-            except astroid.exceptions.StatementMissing:
-                continue
+                attr_nodes = owner.getattr(node.attrname)
             except AttributeError:
```

Comment:
I believe this is already uncovered due to the upgrade. Can you remove it?

Reply:
I tried, but had to revert:
```python
AttributeError: 'Lambda' object has no attribute 'getattr'
```

I was planning to have a look this morning at the cause in `astroid`.

Reply:
Strange how it is not covered then. I'll await your investigation 😄

Reply:
Do the primer tests count toward coverage? That would explain it, if not.

Reply:
Ah no, they don't!